src/js/components/RangeSelector/__tests__/RangeSelector-test.js

```diff
@@ -209,4 +209,40 @@ describe('RangeSelector', () => {
     fireEvent.mouseUp(document);
     expect(onChange).toBeCalled();
   });
+
+  test('handle touch gestures', () => {
+    const onChange = jest.fn();
+    const { container, getByLabelText } = render(
+      <Grommet>
+        <RangeSelector values={[10, 20]} onChange={onChange} />
+      </Grommet>,
+    );
+    const rangeContainer = container.firstChild.firstChild;
+
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
Should the `expect(container.firstChild).toMatchSnapshot();` go after the fireEvent stuff at the end of the test? Otherwise not sure what the snapshot is really capturing since it is being taken before any of the touch events happen.

Reply:
hmm yeah, that makes more sense